packages/forms/resources/views/components/tabs.blade.php

```diff
@@ -60,16 +60,18 @@
                 $iconPosition = $tab->getIconPosition();
                 $iconColor = $tab->getIconColor();
 
-                $iconColorClasses = match ($iconColor) {
-                    'danger' => \Illuminate\Support\Arr::toCssClasses(['text-danger-700', 'dark:text-danger-500' => config('tables.dark_mode')]),
-                    'primary' => \Illuminate\Support\Arr::toCssClasses(['text-primary-700', 'dark:text-primary-500' => config('tables.dark_mode')]),
-                    'success' => \Illuminate\Support\Arr::toCssClasses(['text-success-700', 'dark:text-success-500' => config('tables.dark_mode')]),
-                    'warning' => \Illuminate\Support\Arr::toCssClasses(['text-warning-700', 'dark:text-warning-500' => config('tables.dark_mode')]),
-                    'secondary' => \Illuminate\Support\Arr::toCssClasses(['text-gray-700', 'dark:text-gray-300' => config('tables.dark_mode')]),
-                    default => $iconColor,
-                };
+                $iconColorClasses = \Illuminate\Support\Arr::toCssClasses([
+                    ...match ($iconColor) {
```

Comment:
Filament still supports Laravel 8 (which means PHP 8.0) and PHP 8.0 does not support array unpacked yet. So I don't think using array unpacked right here is a good idea.

Reply:
Whoops, good catch! We just decided to support 8.1 for v3, so that's why I accidentally used it here. 😅